Functions/Assertions/PesterThrow.ps1

```diff
@@ -51,7 +51,7 @@ function PesterThrow([scriptblock] $ActualValue, $ExpectedMessage, $ErrorId, [sw
 
 function Get-DoMessagesMatch($ActualValue, $ExpectedMessage, $ExpectedErrorId) {
     if ($ExpectedMessage -eq "") { return $false }
-    return $ActualValue.Contains($ExpectedMessage)
+    return $ActualValue -like "*$ExpectedMessage*"
```

Comment:
Or `return [string]::Equals([string]$ActualValue, [string]$ExpectedMessage, [System.StringComparison]::InvariantCultureIgnoreCase)`.  Lots of ways to skin that particular cat.  :)